bool post_processed
bool gaussian_blur
---
vcpd/Volume v